egs/rm/s5/local/chain/tuning/run_tdnn_wsj_rm_1b.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+# _1b is as _1a, but different as follows
```

Comment:
I think it is good to keep it. Just in cases that chain model doesn't give good alignments as gmm model and it may need tuning left and right tolerance.